### PR TITLE
Update metabolabpy to 0.9.41

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`metabolabpy`](https://bioconda.github.io/recipes/metabolabpy/README.html): **0.6.53** &rarr; **0.9.41**

[install with bioconda](http://bioconda.github.io/recipes/metabolabpy/README.html) [Conda](https://anaconda.org/bioconda/metabolabpy/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/metabolabpy`](https://github.com//bioconda/bioconda-recipes/tree/bump/metabolabpy/recipes/metabolabpy) (click to view/edit other files)
Summary | Python package to process 1D and 2D NMR spectroscopic data for metabolomics and tracer-based metabolism analysis.
Home | [https://github.com/ludwigc/metabolabpy](https://github.com/ludwigc/metabolabpy)
Releases |[https://pypi.org/pypi/metabolabpy/json](https://pypi.org/pypi/metabolabpy/json)
Recipe Maintainer(s) | @RJMW, @ludwigc

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by priesgo** [comment]
Python version needs to be changed to 3.9

**Comment by priesgo** [comment]
ping @RJMW, @ludwigc

**Comment by mencian** [comment]
Requires `pygamma`, which is not yet in conda-forge.

File: recipes/metabolabpy/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
-    - pyside2
+    - pandas
+    - numba
+    - scikit-learn
     - openpyxl
+    - xlsxwriter
+    - multiprocess
+    - pybaselines
+    - darkdetect
+    - pygamma
```
Comment (martin-g) [inline recipes/metabolabpy/meta.yaml:39]:
```suggestion
#    - pygamma  # it is not available neither in conda-forge nor in bioconda channels
```

**Pull request merged**